data/governance.yml

```
@@ -0,0 +1,522 @@
+      link: https://meet.google.com/bwy-fyqo-dnf
+      invite: https://calendar.google.com/calendar/u/4?cid=Y19iMDA5ZDA4MDg0YzIwYWMzMDQ4NjJhN2FkZWJiYjdmOTU0NGIxYzEwMmU0MDMyMTAzMDFiY2ZhYjcwMDBmMjNlQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20
+      notes: https://github.com/ocaml-ppx/ppxlib/wiki
+    members:
+    - name: Sonja Heinze
+      github: pitag-ha
+      role: Lead Maintainer
+    - name: Carl Eastlund
+      github: ceastlund
+      role: Maintainer
+    - name: "Paul-Elliot Angl\xE8s d'Auriac"
+      github: panglesd
+      role: Maintainer
+  - id: utop-dev
+    name: Utop
+    description: Universal toplevel for OCaml
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml-community/utop
+      kind: github
+    members:
+    - name: Etienne Millon
+      github: emillon
+      role: Lead Maintainer
+    - name: Rudi Grinberg
+      github: rgrinberg
+      role: Maintainer
+  - id: opam-publish-dev
+    name: Opam-publish
+    description: A tool to ease contributions to opam repositories
+    contacts:
+    - name: GitHub
+      link: https://github.com/ocaml-opam/opam-publish
+      kind: github
+    members:
+    - name: Louis Gesbert
+      github: AltGr
+      role: Maintainer
+    - name: Raja Boujbel
+      github: rjbou
+      role: Maintainer
+    - name: Kate
+      github: kit-ty-kate
+      role: Maintainer
+    - name: Thomas Gazagnaire
+      github: samoht
+      role: Maintainer
+  - id: ocamlformat-dev
+    name: Ocamlformat
+    description: Auto-formatter for OCaml code
+    contacts:
+    - name: Mailing List
+      link: https://lists.ocaml.org/listinfo/ocamlformat-dev
+      kind: email
+    - name: GitHub
+      link: https://github.com/ocaml-ppx/ocamlformat
+      kind: github
+    members:
+    - name: Guillaume Petiot
+      github: gpetiot
+      role: Lead Maintainer
+    - name: Jules Aguillon
+      github: Julow
+      role: Maintainer
+    - name: Emile Trotignon
+      github: EmileTrotignon
+      role: Maintainer
+  - id: dune-release-dev
+    name: Dune-release
+    description: Streamlining the release of dune packages to opam
+    contacts:
+    - name: GitHub
+      link: https://github.com/tarides/dune-release
+      kind: github
+    members:
+    - name: Etienne Millon
+      github: emillon
+      role: Maintainer
+    - name: Marek Kubica
+      github: Leonidas-from-XIV
+      role: Maintainer
+    - name: Thomas Gazagnaire
+      github: samoht
+      role: Maintainer
+  - id: mdx-dev
+    name: Mdx
+    description: Execute code blocks inside your documentation
+    contacts:
+    - name: GitHub
+      link: https://github.com/realworldocaml/mdx
+      kind: github
+    members:
+    - name: Marek Kubica
+      github: Leonidas-from-XIV
+      role: Lead Maintainer
+    - name: Thomas Gazagnaire
+      github: samoht
+      role: Maintainer
+  - id: ocp-indent-dev
+    name: Ocp-indent
+    description: Indentation tool for OCaml, to be used from editors like Emacs and Vim.
```

```suggestion
    description: Indentation tool for OCaml to be used with editors like Emacs and Vim.
```
Should this be “from” or “with” editors?